Common/src/grid_movement_structure.cpp

```diff
@@ -7755,7 +7815,16 @@ void CSurfaceMovement::WriteFFDInfo(CGeometry *geometry, CConfig *config) {
       output_file << "FFD_DEGREE_I= " << FFDBox[iFFDBox]->GetlOrder()-1 << endl;
       output_file << "FFD_DEGREE_J= " << FFDBox[iFFDBox]->GetmOrder()-1 << endl;
       if (nDim == 3) output_file << "FFD_DEGREE_K= " << FFDBox[iFFDBox]->GetnOrder()-1 << endl;
-      
+      if (config->GetFFD_Blending() == BSPLINE_UNIFORM) {
```

Comment:
Great. Is it possible to use Bezier as the default if there is no FFD_BLENDING option present, while still printing a warning, so that old meshes are still compatible too? Or will this not work with the new changes?